shaku_axum/src/inject_component.rs

```diff
@@ -54,7 +53,7 @@ use std::sync::Arc;
 ///
 ///     let app = Router::new()
 ///         .route("/", get(hello))
-///         .layer(Extension(module));
+///         .with_state(module);
```

Comment:
Now that the mechanism has changed, the description at the top of this documentation should be updated to say there must be an `impl FromRef<S> for Arc<M>`.